Scripts/LookupCSV/LookupCSV.yml

```diff
@@ -0,0 +1,31 @@
+name: LookupCSV
+script: '-'
+type: python
+tags:
+  - file
+  - csv
+  - Utility
+comment: Parses a CSV and looks for a specific value in a specific column, returning
+  a dict of the entire matching row.
+enabled: true
+args:
+  - name: entryID
+    required: true
+    description: EntryID of CSV file.
+  - name: header_row
+    description: CSV file has a header row
+  - name: column
+    description: Column to search for value in
```

Comment:
Should be stated that this could be an integer when no headers, and column name otherwise

Reply:
Good idea, have updated this description.